lighthouse-plugin-publisher-ads/audits/loads-gpt-from-sgdn.js

```diff
@@ -59,11 +59,12 @@ class LoadsGptFromSgdn extends Audit {
   static async audit(artifacts, context) {
     const devtoolsLog = artifacts.devtoolsLogs[Audit.DEFAULT_PASS];
     const networkRecords = await NetworkRecords.request(devtoolsLog, context);
-    const gptUrl = networkRecords.map((r) => new URL(r.url)).find(isGptTag);
-    if (!gptUrl) {
+    const tagUrl = networkRecords.map((r) =>
+      new URL(r.url)).find(isGptOrAMPTag);
+    if (!tagUrl) {
```

Comment:
This audit is specific to GPT so we should revert the changes in this file.

On pages without GPT, the audit would be marked as N/A

Reply:
Thanks for clarifying, I'll remove this and give it a test. I might have changed it because I was seeing a "No GPT" warning when running the reports. It makes sense that this would only show when GPT is precent though, I'll double check that.

Reply:
removed

Reply:
👍 removed this change